files/commands/cron.py

```diff
@@ -87,52 +87,54 @@ def _run_tasks(db_session_factory: scoped_session):
 	running task does not lock the entire table for its entire run, which would
 	for example, prevent any statistics about status from being gathered.
 	'''
-	db: Session = db_session_factory()
```

Comment:
this does different things than what we want. the issue is that the logging seemed to be moved in 4552b817f. the way to get correct session behavior is to disuse `scoped_session` entirely and just make `db_session` a `sessionmaker` (pretty much same interface, but better for more explicit state).

the `with` block just hides the underlying issue here.